qiskit-ibm-runtime-mcp-server/README.md

```diff
@@ -70,12 +73,25 @@ This project recommends using [uv](https://astral.sh/uv) for virtual environment
    await setup_ibm_quantum_account(token="your_token_here")
    ```
 
-   **Token Resolution Priority:**
+   **Credential Resolution Priority:**
    The server looks for credentials in this order:
    1. Explicit token passed to `setup_ibm_quantum_account()`
    2. `QISKIT_IBM_TOKEN` environment variable
    3. Saved credentials in `~/.qiskit/qiskit-ibm.json`
 
+   **Instance Configuration (Optional):**
+   To speed up service initialization, you can specify your IBM Quantum instance:
+   - Set `QISKIT_IBM_RUNTIME_MCP_INSTANCE` environment variable with your instance CRN
+   - This skips the automatic instance lookup which can be slow
+   - Find your instance CRN in [IBM Quantum Platform](https://quantum.cloud.ibm.com/)
```

Comment:
would it make more sense to direct users to the instances page:
https://quantum.cloud.ibm.com/instances